pkg/commands/clusterstack/update_test.go

```diff
@@ -422,8 +420,6 @@ Build and Run images already exist in stack
 	})
 
 	when("dry-run flag is used", func() {
-		fakeRelocator.SetSkip(true)
```

Comment:
Doing this was masking that the wrong "skip" behavior was being passed to the relocator on dry-run-with-image-upload